api/src/opentrons/calibration_storage/encoder_decoder.py

```diff
@@ -3,25 +3,26 @@
 """
 import json
 import datetime
+from typing import Any
 
 
 class DateTimeEncoder(json.JSONEncoder):
-    def default(self, obj):
+    def default(self, obj: Any) -> Any:
```

Comment:
```suggestion
    def default(self, obj: object) -> Any:
```

This doesn't *really* matter, but it will prevent the body of this method from doing `obj.foo()` outside of an `isinstance()` guard that guarantees `foo()` will be present. https://stackoverflow.com/questions/39817081/typing-any-vs-object